llvm.proj

```diff
@@ -89,8 +89,9 @@
     <_LLVMBuildArgs Condition="'$(BuildOS)' == 'Windows_NT' and '$(Configuration)' == 'Release'" Include='-DLLVM_USE_CRT_RELEASE=MT' />
     <_LLVMBuildArgs Condition="'$(BuildOS)' == 'Windows_NT' and '$(Configuration)' == 'Debug'" Include='-DLLVM_USE_CRT_DEBUG=MTd' />
     <_LLVMBuildArgs Condition="'$(BuildOS)' == 'Windows_NT' and '$(Configuration)' == 'Debug'" Include='-DLLVM_USE_CRT_RELEASE=MTd' />
-    <_LLVMBuildArgs Condition="'$(BuildOS)' == 'Windows_NT'" Include='-DLLVM_ENABLE_ZLIB=OFF' />
     <_LLVMBuildArgs Condition="'$(BuildOS)' == 'Windows_NT'" Include='-DLLVM_NATIVE_TOOL_DIR=$(NativeTablegenDir)' />
+    <_LLVMBuildArgs Include='-DLLVM_ENABLE_ZLIB=FORCE_ON' />
```

Comment:
https://github.com/dotnet/llvm-project/blob/60d59a87f949ba5ee7d521d9d4fd523b5f874d68/llvm/CMakeLists.txt#L493

FORCE_ON implies if zlib was not present on the build machine, it will fail at config time.

Reply:
Would it be nicer to condition this to "not Windows"?

Reply:
It is there at the line below, but they may not cancel each other, lets add the explicit condition:
```suggestion
    <_LLVMBuildArgs Condition="'$(BuildOS)' != 'Windows_NT'" Include='-DLLVM_ENABLE_ZLIB=FORCE_ON' />
```